### PR TITLE
patch appears under library on click, fix

### PR DESCRIPTION
**Pull request by imgrby**

### CODE REVIEW

**Comment by kschzt** [comment]
otherwise good, but now it blinks:
patchlibhide

File: browser/scripts/collapsible-select-control.js [hunk 1]
```diff
@@ -134,7 +141,12 @@ function dragAndDropMouseDownHandler(e) {
 		mouseMoveBound = false
 		scrollBound = false
 		clearInterval(scrollInterval)
-
+		
+		// Hide  source library if patch appeared under it after click
+		if(evt.pageX < (parentWidth + parentX) && evt.pageX > parentX && evt.pageY < (parentHeight + parentY) && evt.pageY > parentY) {
+			itemParent.fadeOut().delay(1000).fadeIn();
```
Comment (gmarinov) [inline browser/scripts/collapsible-select-control.js]: please make sure the state of these things is updated at all times if you're modifying them
// in VizorUI
this.visibility = {			// granular flags
	...
	panel_chat: true,
	panel_assets: true,
	panel_presets: true,
	patch_editor: true,
	breadcrumb: true,
	player_controls : true,
	main_toolbar : true
};

**Comment by imgrby** [comment]
interesting

**Comment by imgrby** [comment]
fixed so now it just closes floating box if dropped under it, expecting user is focused on graph not ui

**Comment by kschzt** [comment]
ok lgtm

**Pull request merged**